config/application.rb

```diff
@@ -25,5 +25,12 @@ class Application < Rails::Application
     # -- all .rb files in that directory are automatically loaded.
     # Load the integrations
     config.autoload_paths << Rails.root.join('lib', 'integrations')
+
+    config.middleware.insert_before 0, Rack::Cors do
```

Comment:
Probably not. Try it without it. If it does need it, only allow it in the single route you need it for